torch/_prims/utils.py

```diff
@@ -23,12 +23,17 @@
         torch.long: DataType.Int,
         torch.int: DataType.Int32,
         torch.bool: DataType.Bool,
+        # Python scalars
+        complex: DataType.ComplexDouble,
+        float: DataType.Double,
+        int: DataType.Int,
+        bool: DataType.Bool,
     }
 else:
     _torch_dtype_to_nvfuser_dtype_map = {}
 
 
-def getnvFuserDtype(dtype: torch.dtype):
+def getnvFuserDtype(dtype: Union[torch.dtype, type]):
```

Comment:
Question: Why is this a union?  I am not sure what the other `type` is.

Reply:
This function is now used to get the nvFuser's type for Python scalars (float, complex, bool, int) and the type of float/complex/bool/int is `type`.

Reply:
Thanks!